docs/requirements.txt

```diff
@@ -1,17 +1,9 @@
-# Requirements list for WEIS
-
-# simulation
-matplotlib
-numpy
-scipy
-openmdao>=3.2
-simpy
-marmot-agents
-
-# docs
-sphinx>2.0
-numpydoc
-numpydoc
-sphinxcontrib-bibtex<2.0.0
-sphinx-rtd-theme
-# sphinx-autoapi
\ No newline at end of file
+breathe>=4.4

```

Comment:
Are these changes purposeful too? It looks like they don't include a few that are required for WEIS -- like OpenMDAO.